.pre-commit-config.yaml

```diff
@@ -33,12 +33,11 @@ repos:
   rev: 21.12b0
   hooks:
   - id: black
-    args: [--target-version=py37]
 
-- repo: https://github.com/PyCQA/flake8
-  rev: 4.0.1
+- repo: https://github.com/csachs/pyproject-flake8
+  rev: v0.0.1a2.post1
   hooks:
-  - id: flake8
+  - id: pyproject-flake8
```

Comment:
FYI, Anthony and co. actively fight others adding pyproject.toml support as well, see https://github.com/csachs/pyproject-flake8/issues/13. Currently broken with Flake8 5 though might be fixable for now.